app/src/main/java/com/sopt/dive/presentation/home/HomeScreen.kt

```diff
@@ -9,36 +9,40 @@ import androidx.compose.foundation.lazy.LazyColumn
 import androidx.compose.foundation.lazy.itemsIndexed
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
-import androidx.compose.runtime.remember
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.lifecycle.viewmodel.compose.viewModel
 import com.sopt.dive.R
 import com.sopt.dive.core.designsystem.theme.DiveTheme
-import com.sopt.dive.data.local.UserPreferences
-import com.sopt.dive.domain.model.friend.dummyFriendProfiles
+import com.sopt.dive.domain.model.friend.FriendProfile
+import com.sopt.dive.presentation.home.HomeViewModel.Companion.dummyFriendProfiles
 import com.sopt.dive.presentation.home.component.FriendCard
 import com.sopt.dive.presentation.home.component.MyProfileCard
+import kotlinx.collections.immutable.ImmutableList
 
 @Composable
 fun HomeRoute(
-    paddingValues: PaddingValues
+    paddingValues: PaddingValues,
+    viewModel: HomeViewModel = viewModel()
 ) {
-    val context = LocalContext.current
-    val userPreferences = remember { UserPreferences(context) }
+    val userInfo = viewModel.myProfile.collectAsStateWithLifecycle().value
```

Comment:
```suggestion
    val userInfo by viewModel.myProfile.collectAsStateWithLifecycle()
```

여기서는 코틀린의 **Deligation**을 사용하는게 좋습니다ㅎㅎ
cc. @ckals413 @dmp100

Reply:
훨씬 깔끔하고 명확해지겠네요ㅎㅎ
짚어주셔서 감사합니다! 수정하겠습니다